frontend/src/pages/ConnectedPages/Game/index.tsx

```diff
@@ -15,24 +14,29 @@ import ConnectionManager from "../../../websockets/ConnectionManager";
 import Question from "./Question";
 import { getToken } from "../../../utils/cookies";
 import AnswerResult from "./AnswerResult";
+import { adjustVolume } from "../../../utils/audio";
 
 const Game: React.FC<RouteComponentProps> = ({ location, history }) => {
   const [step, setStep] = useState<IGameEnum>(IGameEnum.COUNTDOWN);
   const [previewUrl, setPreviewUrl] = useState<string>("");
   const [answers, setAnswers] = useState<Answer[]>([]);
   const [isAnswerCorrect, setIsAnswerCorrect] = useState<boolean>(false);
-  const [score] = useState<number>(0);
+  const [isMuted, setIsMuted] = useState<boolean>(false);
+  // const [score] = useState<number>(0);
   const player = useRef<null | HTMLAudioElement>(null);
 
   useEffect(() => {
-    reloadPlayer();
+    reloadPlayer().catch();
   }, [previewUrl]);
 
-  const reloadPlayer = async () => {
+  const reloadPlayer = async (): Promise<void> => {
     if (player && player.current) {
+      await adjustVolume(player.current, 0, { duration: 200 });
```

Comment:
I am afraid this would lead to hear the end of the previous song at the beginning of the next question.
Is it the case ?

Reply:
No the promise in adjustVolume wait for the duration before resolving.